sloglint.go

```diff
@@ -376,6 +376,8 @@ func isStaticMsg(msg ast.Expr) bool {
 		return msg.Kind == token.STRING
 	case *ast.Ident: // e.g. const msg = "msg"; slog.Info(msg)
 		return msg.Obj != nil && msg.Obj.Kind == ast.Con
+	case *ast.BinaryExpr: // e.g. slog.Info("x" + "y")
```

Comment:
Don't you need to check the operator?

https://pkg.go.dev/go/ast#BinaryExpr

Reply:
Yep, only `+` can be applied to strings. Added an assertion just in case